### DISCUSSION TITLE
Verification email for GitHub student pack not received

### DISCUSSION THREAD

**Discussion by decten** [community/community#106633] [GitHub Education]
### Hi everyone,

I applied for the GitHub student pack with my school email, but I have not received the verification email from GitHub. 
I checked my spam and junk folders, verified my email address, and tried to resend the verification email, but nothing worked. 
I also noticed that there is no resend button on the application page. 
![그림1](https://github.com/community/community/assets/52107658/d528e2ed-ddc3-4972-ac90-60a18346a525)
I have previously verified my school email with GitHub, so I don’t know why I can’t receive the verification email now. 
Can anyone help me with this issue?😢 
Thank you

**Comment by Anixton** [comment]
Did you check your school mail, not personal mail just to make sure?
If you don't see contact github support they handle these kind of things.

If this was helpful please mark this comment as answer!

**Reply by decten** [reply]
Yes, I checked my school email, not my personal email. It worked fine with jetbrains and I downloaded IntelliJ using the same school email an hour ago
I also contacted GitHub support about this issue, and I received a confirmation email to my school emailㅠㅠ

Thank you for your reply. I appreciate your help😊

**Comment by samus-aran** [comment]
Hey @decten  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

If you don't receive your benefits within 5 days of your approved status please feel free to re-open this discussion 👍

**Reply by decten** [reply]
The problem is not the education benefits, but the fact that I still haven’t completed the verification because the verification email hasn’t arrived even now, 3 days later.

**Reply by decten** [reply]
<img width="959" alt="image" src="https://github.com/community/community/assets/52107658/3976cd42-1954-4c87-8ac5-144aa0a8f970">
<img width="686" alt="image" src="https://github.com/community/community/assets/52107658/ff997f4a-67bb-4b67-b3ad-f74f43fe4015">

I only received the email that said “two-factor authentication factor configured”, but I didn’t receive the verification email again after that😢

**Reply by samus-aran** [reply]
You won't until you receive your academic benefits which you should receive after you have been approved.

Your educational benefits will arrive within 72 hours of your "approved" status. **However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll then get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application).

It has been 3 days so I suspect you'll receive this email ☝️ soon.
